kube/src/lib.rs

```diff
@@ -76,31 +76,55 @@
 
 #[macro_use] extern crate static_assertions;
 assert_cfg!(
-    all(
-        not(all(feature = "native-tls", feature = "rustls-tls")),
-        any(feature = "native-tls", feature = "rustls-tls")
-    ),
+    not(all(feature = "native-tls", feature = "rustls-tls")),
     "Must use exactly one of native-tls or rustls-tls features"
 );
```

Comment:
We should keep the static assertion ensuring a TLS stack is specified when `client` is enabled. `--no-default-features --features ws` for example will show a confusing error about the use of undeclared module `connector`.

Reply:
ah, of course. put that back.